apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -184,12 +184,19 @@ <h3 class="ledgerHead">
                     </thead>
                     <tbody>
                         <tr>
-                            <td class="nopad bdrL bdrR hidden-xs" style="width:50%" colspan="3">
+                            <td width="100%" colspan="100%" class="ledger-head transction-mapped">
+                                <table class="table" width="100%">
+                                    <tr>
+                                        <th colspan="6">Debit <span class="pull-right text-gray">Dr</span></th>
+                                        <th colspan="6">Credit <span class="pull-right text-gray">Cr</span></th>
+                                    </tr>
+                                </table>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="nopad bdrL bdrR hidden-xs transction-mapped-head" style="width:50%" colspan="3">
```

Comment:
Avoid incorrect spelling in class names
Avoid inline styles

Reply:
Done
Inline css written already , i am not added this